test/util.go

```diff
@@ -32,5 +32,5 @@ func LogResourceObject(logger *logging.BaseLogger, value ResourceObjects) {
 
 // ImagePath is a helper function to prefix image name with repo and suffix with tag
 func ImagePath(name string) string {
-	return fmt.Sprintf("%s/%s:%s", ServingFlags.DockerRepo, name, ServingFlags.Tag)
+	return fmt.Sprintf("%s:test-%s", ServingFlags.DockerRepo, name)
```

Comment:
We need to find a better way to change the image path for OpenShift.

Reply:
The `test-` is specific to test images (as opposed to the main knative images). See the configuratino of ci-operator at https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-serving/openshift-knative-serving-master.yaml